consensus/types/src/slot_epoch_macros.rs

```diff
@@ -290,8 +290,8 @@ macro_rules! impl_ssz {
                 tree_hash::TreeHashType::Basic
             }
 
-            fn tree_hash_packed_encoding(&self) -> Vec<u8> {
-                ssz_encode(self)
+            fn tree_hash_packed_encoding(&self) -> tree_hash::PackedEncoding {
+                tree_hash::PackedEncoding::from_slice(&self.0.to_le_bytes())
```

Comment:
Could defer to the `u64` impl here (reduces the diff relative to `tree-states` :grin: )

```suggestion
                self.0.tree_hash_packed_encoding()
```

Reply:
Good call!